index.html

```diff
@@ -207,6 +199,81 @@ <h3 class="skill-name">Skills</h3>
           </div>
         </div>
       </section>
+      <footer class="contact-form">
+        <div class="chat">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
+        </div>
+        <form action="https://formspree.io/f/mbjboewz" method="post">
+          <ul class="form">
+            <li>
+              <input
+                type="text"
+                id="name"
+                name="user_name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
+            </li>
+            <li>
+              <input
+                name="user_email"
+                type="email"
+                id="email"
+                placeholder="Email address"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="message"
+                id="textarea"
+                placeholder="Enter text here"
+                maxlength="500"
+                required
+              ></textarea>
+            </li>
+            <li>
+              <button class="btn" type="submit">Get in touch</button>
+            </li>
+          </ul>
+        </form>
```

Comment:
[OPTIONAL] It's better to move the contact section outside the footer since it's not a part of the footer 🤗

Reply:
@awais-amjed Thank you for your great review.😀 I put it inside the footer due to the requirement of the project. Please see the requirement below.

The section you need to build for this milestone is:
the contact form in the footer:

Thank you